src/app/phase-tester-response/issue-faulty/issue-faulty.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, OnInit, ViewChild } from '@angular/core';
+import { Issue } from '../../core/models/issue.model';
+import { TABLE_COLUMNS } from '../../shared/issue-tables/issue-tables-columns';
+import { ACTION_BUTTONS, IssueTablesComponent } from '../../shared/issue-tables/issue-tables.component';
+
+@Component({
+  selector: 'app-issue-faulty',
+  templateUrl: './issue-faulty.component.html',
+  styleUrls: ['./issue-faulty.component.css']
+})
+export class IssueFaultyComponent implements OnInit {
+  @ViewChild(IssueTablesComponent, { static: true }) table: IssueTablesComponent;
```

Comment:
Since ViewChild has a default flag of static:false, isit possible to make use of AfterViewInit rather than OnInit?

Reply:
I think #867 tried to do that for the rest of the issue tables but there's some issues in doing so :( See also the discussion at #773